examples/recsys2020.ipynb

```diff
@@ -300,76 +462,131 @@
-    "        op_name = 'wd',\n",
-    "        f = lambda col, gdf: cudf.to_datetime(col, unit='s').dt.weekday,\n",
-    "        columns = ['timestamp'],\n",
-    "        replace=False\n",
-    "    ),\n",
-    "    # Count the number of tokens in the columns hashtags, domains, links\n",
+    "proc.add_feature(\n",
     "    nvt.ops.LambdaOp(\n",
     "        op_name='count_t',\n",
     "        f=lambda col, gdf: count_token(col,'\\t'),\n",
     "        columns=['hashtags', 'domains', 'links'],\n",
     "        replace=False\n",
-    "    ),\n",
-    "    # Fill in na/missing values in target columns + hashtags, domains, links\n",
-    "    nvt.ops.FillMissing(columns=label_name + ['hashtags', 'domains', 'links']),\n",
-    "    # Change data type to uint32 to save memory/storage\n",
-    "    nvt.ops.LambdaOp(\n",
-    "        op_name='astypeint32',\n",
-    "        f=lambda col, gdf: col.astype(np.uint32),\n",
-    "        columns=label_name + ['timestamp',\n",
-    "                              'a_follower_count', \n",
-    "                              'a_following_count',\n",
-    "                              'a_account_creation',\n",
-    "                              'b_follower_count',\n",
-    "                              'b_following_count',\n",
-    "                              'b_account_creation'],\n",
-    "        replace=True\n",
-    "    ),\n",
-    "    # Apply splitmedia function\n",
+    "    )\n",
+    ")"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "We apply splitmedia function to split the media."
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": 18,
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "proc.add_feature(\n",
     "    nvt.ops.LambdaOp(\n",
     "        op_name='splitmedia',\n",
-    "        f=lambda col, gdf: splitmedia2(col),\n",
+    "        f=lambda col, gdf: splitmedia(col),\n",
     "        columns=['media'],\n",
     "        replace=True\n",
-    "    ),\n",
-    "    # Encode columns in continous integer to save memory\n",
+    "    )\n",
+    ")"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "We fill in na/missing values in target columns + hashtags, domains, links."
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": 19,
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "proc.add_feature(\n",
+    "    nvt.ops.FillMissing(columns=label_name + ['hashtags', 'domains', 'links'])\n",
+    ")"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "We encode columns in continous integer to save memory. Some categorical columns contain long hashes of type String as values to preserve the privacy of the users (e.g. userId, language, etc.). Long hashes of type String requires signficant amount of memory to store. We encode/map the Strings to continous Integer to save significant memory. Finally, we optimize the datatype."
```

Comment:
A couple typos here . Also do we really want to go with 'continuous integer' to describe our categorical encoding? Maybe we should describe as 'small contiguous integers' instead?

```suggestion
    "We encode columns as a continuous integer to save memory. Some categorical columns contain long hashes of type String as values to preserve the privacy of the users (e.g. userId, language, etc.). Long hashes of type String requires significant amount of memory to store. We encode/map the Strings to continuous Integer to save significant memory. Finally, we optimize the datatype."
```

Reply:
Thanks, I added 'small' to it. I think the gist is that long Strings are inefficient to represent categorical values. I hope this is communicated.